.gitignore

```diff
@@ -1,7 +1,9 @@
 *.pyc
 *.pyo
-plugincheck/settings_local.py
+plugincheck/settings/local.py
+plugincheck/settings/test.py
```

Comment:
Why ignore test? I would think it's nice to have `test.py` in here since you can control it and thus everyone knows what everyone else is testing with.